src/pretix/base/forms/questions.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, scheme: dict, field: forms.Field, attrs=None, titles: list=No
             a['data-fname'] = fname
             if fname == 'title' and self.titles:
                 widgets.append(Select(attrs=a, choices=[('', '')] + [(d, d) for d in self.titles[1]]))
+            elif fname == 'salutation':
+                widgets.append(Select(attrs=a, choices=[('', '---')] + [(s, s) for s in PERSON_NAME_SALUTATIONS]))
```

Comment:
I put the `---` to indicate that a choice *must* be made. The title doesn't have it, as it can be left empty.

I just noticed that validation should therefore be different then.

Reply:
Ah nevermind there is `REQUIRED_NAME_PARTS` :+1: